config/routes.rb

```diff
@@ -1,11 +1,16 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
-  root "microposts#index"
+  root "static_pages#home"
   
   resources :microposts, only: %i[index create destroy]
-  resources :users, only: %i[index show new create edit update destroy]
+  resources :users, only: %i[index show new create edit update destroy] do
```

Comment:
```suggestion
  resources :users do
```
Đủ 7 action rồi thì e bỏ only đi nhé